.github/ISSUE_TEMPLATE/INTERNAL.yml

```diff
@@ -0,0 +1,39 @@
+name: Team Internal
+description: Issue template used for tasks created by the NuGet team for themselves. Internal Use Only.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Preflight Checklist
+
+        **Do not file this issue until you've gone through this checklist.**
+
+        ### Basic Labels (required)
+
+        * Add **Regression** labels if this is a regression.
+        * Add a **Type:** label
```

Comment:
I think right now, it's either `Type` or `Category:Engineering`. 

I'd prefer to have had Type:Engineering, but I can't make that call :)

Reply:
We're having the `Type:Engineering` discussion in https://teams.microsoft.com/l/message/19:00d019d741b44e54baadd67ef7aeb41d@thread.skype/1634250158470?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ad34ac55-6785-4e7c-82c3-7ba405d30c3c&parentMessageId=1634250158470&teamName=NuGet&channelName=Client&createdTime=1634250158470